client/src/components/Pill/Pill.css

```diff
@@ -124,6 +124,13 @@
   font-size: 24px;
 }
 
+.Pill--subnav {
+  /* padding: 0.42em 3.35em; */
+  font-size: 36px;
+  width: 50vw;
```

Comment:
It would be better to use rem here. vw and vh are good for when you're styling a big piece of the user interface (like the header or footer)